### PR TITLE
AO3-5775 Add language to work meta in downloads

### PR DESCRIPTION
**Pull request by korrien**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open?

## Issue

https://otwarchive.atlassian.net/browse/AO3-5775

## Purpose

Fixes issue AO3-5775. Added test to record existing tags in the header of the downloaded page.
There are some other differences between work view on the website and downloaded page. Noted some of them in the test, didn't attempt to unify or do anything else due to contributing guidelines.

In case I did something totally stupid - sorry, I am completely new to this tech stack and to the project :)  

## Credit

korrien, she/her

### CODE REVIEW

**Review by sarken — CHANGES_REQUESTED** [review]
Hi, korrien!

Thank you so much for contributing! We're always happy to get pull requests from new people. If you think you'll be contributing more, you're welcome to create a Jira account so you can claim issues. If you do, we ask that the "Full name" you provide either matches or contains your GitHub name, since that helps us keep track of who is who. (Once you've made a Jira account, you can reply here with the name and I'll make sure you have the right permissions to claim issues.)

Now onto the code review! The code looks good -- I just had a few small suggestions for changes. There's no rush to make them, though, because unfortunately, this is planned for a dedicated [downloads release](https://otwarchive.atlassian.net/issues/?jql=project%20%3D%20%22AO3%22%20AND%20fixVersion%20%3D%20%220.9.2xx%20(Downloads)%22), which still needs a lot of work. (Whenever we change the format of work downloads, we have to delete existing files from our servers to avoid having inconsistent formatting, so we prefer to do as many as possible in one release.)

Thank you again for contributing!

File: features/works/work_download.feature [hunk 1]
```diff
@@ -51,6 +51,69 @@ Feature: Download a work
   Then I should be able to download all versions of "Emjoi 🤩 Yay 🥳"
 
 
+  Scenario: Downloaded work header contains expected meta fields in expected order
+
+  Given basic tags
+    And I have a collection "My Collection 1" with name "mycollection1"
+    And I have a collection "My Collection 2" with name "mycollection2"
+    And I am logged in
+    And I go to the new work page
```
[the remark this answers is not in the document]
Comment (korrien) [reply features/works/work_download.feature:60]: Thanks for pointing me to that method, haven't seen it before :)

I tried to modify the code to use this one, but decided against it due to decreased readability.
Testing just Language presence and position didn't seem like a good test, so I went for testing all possible meta fields.
As there are many of them, draft step becomes waaaay to long even if I cut some characters in strings. Line doesn't fit to one screen and, as far as I understand, Cucumber doesn't allow to break lines inside the step.
Putting just a couple of things as part of the draft seems strange.

So left as is. But surely will change if my arguments are not convincing enough :)

**Review by korrien — COMMENTED** [review]
Hi sarken :)

Thank you for the review and explanations!
That issue seemed to be the easiest one for the beginner like me, but shall leave Download alone for now.

1. Jira
Account 'korrien' created, as I *hope* that this isn't the last you've seen of me :D 

2. Travis
Some test originally failed, but I will assume this is one of the random Travis failures you mention in the guide.
Locally failed feature works fine.
On my Travis branch either all tests passed or some different test with bylines check failed. This intermittent failure doesn't seem to be caused by my changes.

3. Follow-up commits
Didn't find any guidelines on whether you prefer post-review changes as fix-ups or as force-pushes, so forced-push as PR is small.

Thanks :)

**Review by sarken — APPROVED** [review]
This looks good -- thank you! And thanks for pointing out the discrepancies between downloads and work show pages; I'll make an issue to correct the new ones you noted in the tests.

> Account 'korrien' created, as I hope that this isn't the last you've seen of me :D

Excellent! I've assigned this issue to you and given you permission to claim and comment on others.

> Some test originally failed, but I will assume this is one of the random Travis failures you mention in the guide.

It looks like someone reran the test and it passed, so you are indeed correct!

>Didn't find any guidelines on whether you prefer post-review changes as fix-ups or as force-pushes, so forced-push as PR is small.

Ah, thanks for pointing this out! We'll have to add some info on this to our wiki. We find fixup and force pushes can make the review process a little confusing, so whenever possible, we prefer folks to simply add any post-review changes as new commits and push them.

**Comment by redsummernight** [comment]
This currently has merge conflicts.

**Pull request merged**